airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeCopyAzureBlobStorageDestination.java

```diff
@@ -63,6 +73,47 @@ public SqlOperations getSqlOperations() {
     return new SnowflakeSqlOperations();
   }
 
+  @Override
+  public AirbyteConnectionStatus check(final JsonNode config) {
+    try {
+      checkPersistence(config);
+    } catch (final Exception e) {
+      LOGGER.error("Exception attempting to access the staging persistence: ", e);
+      return new AirbyteConnectionStatus()
+          .withStatus(AirbyteConnectionStatus.Status.FAILED)
+          .withMessage("Could not connect to the staging persistence with the provided configuration. \n" + e.getMessage());
+    }
+
+    final DataSource dataSource = getDataSource(config);
+
+    try {
+      final JdbcDatabase database = getDatabase(dataSource);
+      final var nameTransformer = getNameTransformer();
+      final var outputSchema = nameTransformer.convertStreamName(getConfiguredSchema(config));
+      AbstractJdbcDestination.attemptSQLCreateAndDropTableOperations(outputSchema, database, nameTransformer, getSqlOperations(), true);
+
+      return new AirbyteConnectionStatus().withStatus(AirbyteConnectionStatus.Status.SUCCEEDED);
+    } catch (final ConnectionErrorException ex) {
```

Comment:
cc: @akashkulk since I know you're in the process of deprecating `ConnectionErrorException` class to become `ConfigErrorException`. Can you provide some guidance to whether this change should be handled differently?

Reply:
Removed this overridden method at all. Thanks

Reply:
Based on this comment, doesn't seem to be relevant - but in the future you wouldn't need any of the code that catches any of the exceptions and instead throw a `ConfigErrorException`